spec/order_form_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'order_form'
+
+describe OrderForm do
+  
+  it { is_expected.to respond_to :show_dishes }
+  
+  it 'shows dishes' do
+    expect(subject.show_dishes).to eq 'Noodles (£3.00), Rice (£2.50)'
+  end
+
+  it 'adds dishes to the order' do
+    subject.add_dish :Rice, 2
+    expect(subject.order.length).to eq 2
+  end
+
+  it 'does not add dishes that are not on the menu' do
+    expect{ subject.add_dish :Pizza, 1 }.to raise_error 'Not on the menu'
+  end
+
+  it 'errors if nothing ordered' do
+    expect{ subject.check_total 0 }.to raise_error 'No dishes were selected'
+  end
+
+  it 'errors if customer total is incorrect' do
+  	subject.add_dish :Rice, 1
+    expect{subject.check_total 3}.to raise_error 'Error, total incorrect'
```

Comment:
Inconsistent indentation detected.<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.